projects/ad9371/src/app/headless.c

```diff
@@ -314,6 +314,12 @@ int main(void)
 
 	printf("Please wait...\n");
 
+	/* Instruction cache should be enabled for usleep functions to work. */
+	/* Enable the instruction cache. */
+	Xil_ICacheEnable();
```

Comment:
this ad9371 change should probably go in the 2nd commit together with all the other such changes